src/app/shared/modules/routing.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { LoginPageComponent } from 'src/app/pages/login-page/login-page.component';
+import { MainPageComponent } from 'src/app/pages/main-page/main-page.component';
+import { AuthGuard } from '../guards/auth.guard';
+
+const routes: Routes = [
+  { path: '', component: MainPageComponent, canActivate: [AuthGuard] },
+  { path: 'login', component: LoginPageComponent },
+];
```

Comment:
generally, if we have '' route, we redirect to logic, but if user is logged in, we redirect to 'main'/'dashboard' route (it's rarely stays empty)